### PR TITLE
Added dnsoptions to the docker driver

### PR DESCRIPTION
**Pull request by decoomanj**
The DnsOptions for Docker were missing.

### CODE REVIEW

**Comment by decoomanj** [comment]
Weird, first build was successful. Then I fixed a typo in a comment, and the build failed :thinking:

**Comment by decoomanj** [comment]
Great, now it builds again.

File: client/driver/docker_test.go [hunk 1]
```diff
@@ -879,6 +880,10 @@ func TestDockerDriver_DNS(t *testing.T) {
 	if !reflect.DeepEqual(task.Config["dns_search_domains"], container.HostConfig.DNSSearch) {
 		t.Errorf("DNS Servers don't match.\nExpected:\n%s\nGot:\n%s\n", task.Config["dns_search_domains"], container.HostConfig.DNSSearch)
 	}
+
+	if !reflect.DeepEqual(task.Config["dns_options"], container.HostConfig.DNSOptions) {
+		t.Errorf("DNS Servers don't match.\nExpected:\n%s\nGot:\n%s\n", task.Config["dns_options"], container.HostConfig.DNSOptions)
```
Comment (ctlajoie) [inline client/driver/docker_test.go]: Hardly worth mentioning but looks like there's a copy/paste bug in there. Probably want to change "Servers" to "Options".

Comment (decoomanj) [inline client/driver/docker_test.go]: Done! Also updated the text in the statement above for the search_domains which was wrong too.

**Comment by decoomanj** [comment]
Again CI doesn't build because of other reasons :-(

**Comment by decoomanj** [comment]
PR doesn't build because of missing AWS stuff??

**Comment by dadgar** [comment]
Thanks @decoomanj! Yeah build is breaking because a PR from the other day. Will resolve soon!

**Pull request merged**